components/runtime/smalltalk/squeak/Makefile

```diff
@@ -43,6 +43,10 @@
 BUILD_BITS=32_and_64
 USE_COMMON_TEST_MASTER=no
```

Comment:
You are still building 32 bit which we phase out at the moment.
Your build won't work anymore because we don't have all needed 32 bit libraries anymore.